documentation/docs/examples/cad-3dobjects.mdx

```diff
@@ -0,0 +1,120 @@
+---
+id: cad-3dobjects
+title: Add custom 3D objects
+description: Extend the scene with custom ThreeJS objects.
+---
+
+import { Water } from 'three/examples/jsm/objects/Water';
+import { LiveCodeSnippet }from '../../src/components/LiveCodeSnippet';
+import { DemoWrapper } from '../../src/components/DemoWrapper';
+import { Cognite3DModel, Cognite3DViewer } from '@cognite/reveal';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Sometimes it's useful to add custom content to the 3D scene to model data Reveal doesn't support or to customize the 3D environment. Reveal supports this by using the `Cognite3DViewer.addObject3D`-function which accepts [ThreeJS](https://threejs.org/) objects. 
+
+:::tip
+Reveal uses an "implicit lighting model", i.e. no lights are added to the scene. If you need lights this needs to be added to the viewer scene. 
+:::
+
+<DemoWrapper modelId={4715379429968321} revisionId={5688854005909501} name="Cognite3DViewerDemo" />
+
+## Adding markers to the scene on clicks
+
+The following examples reacts to clicks in the scene and adds markers to the positions clicked.
```

Comment:
looks like a typo here. should either be "example reacts adds" or "examples react add". Consider using google docs to catch some typos like "lets/let's".

Reply:
typo is still here 

> should either be "example reacts adds" or "examples react add".